packages/features/ee/teams/components/CreateANewTeamForm.tsx

```diff
@@ -16,6 +16,19 @@ const querySchema = z.object({
   returnTo: z.string(),
 });
 
+function noTrailingHyphenSlugify(str: string) {
+  const slugified_str = slugify(str);
```

Comment:
I am not sure what testcase I missed but the objective can be met by 
```javascript
function noTrailingHyphenSlugify2(str) {
  const slugified_str = slugify(str);
  return slugified_str.replace(/-*$/, '')
}`

Reply:
Also, I think we can improve the slugify logic itself to tackle this. No one would want trailing hyphens.